sharding-sql-test/src/main/resources/sql/sharding/dml/insert.xml

```diff
@@ -48,4 +48,7 @@
     <sql-case id="insert_with_null_value" value="INSERT INTO t_null_value_test(col1) VALUES(null)" db-types="MySQL" />
     <sql-case id="insert_with_blob_value" value="INSERT INTO t_blob_value_test(col1) VALUES(_BINARY'This is a binary value.')" db-types="MySQL" />
     <sql-case id="insert_with_function" value="INSERT INTO t_order(current_date, order_id, user_id) VALUES (curdate(), ?, ?)" db-types="MySQL" />
+    <sql-case id="insert_duplicate_key_update_with_function_common" value="INSERT INTO emp(id,age,salary) VALUES (?,?,?) ON DUPLICATE KEY UPDATE salary = salary+1" db-types="MySQL"/>
+    <sql-case id="insert_duplicate_key_update_two_group_data" value="INSERT INTO emp(id,age,salary) VALUES (?,?,?),(?,?,?) ON DUPLICATE KEY UPDATE salary = VALUES(id)+VALUES(age)" db-types="MySQL"/>
```

Comment:
The SQL fragment of `salary = VALUES(id)+VALUES(age)` does not make sense, use `salary = VALUES(salary)+VALUES(salary) + 10` is better